src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/Testing/NUnit/IResultState.cs

```diff
@@ -0,0 +1,31 @@
+// <copyright file="IResultState.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.NUnit
+{
+    /// <summary>
+    /// DuckTyping interface for NUnit.Framework.Interfaces.ResultState
+    /// </summary>
+    public interface IResultState
+    {
+        /// <summary>
+        /// Gets the TestStatus for the test.
+        /// </summary>
+        /// <value>The status.</value>
+        TestStatus Status { get; }
+
+        /// <summary>
+        /// Gets the label under which this test result is
+        /// categorized, or <see cref="string.Empty"/> if none.
+        /// </summary>
+        string Label { get; }
```

Comment:
I don't think this property is used, do you need it?

Reply:
you're right, removing it.

Reply:
done in fd3279a